asteroids_code/BoilerPlate/Asteroids.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <vector>
+#include "vector2.hpp"
+#include "MathUtilities.hpp"
+
+const float asteroid_speed = 1.0f;
+const float asteroid_rotation = 1.0f;
+class Asteroids
+{
+	//functions
+public:
+	Asteroids();
+	Asteroids(Vector2&, float);
+	void Render();
+	void Move();
```

Comment:
I can't seem to understand the need for this function. Maybe you could explain it.